lib/OpenQA/Worker/CommandHandler.pm

```diff
@@ -109,18 +109,9 @@ sub handle_command {
             return $worker->skip_job($job_id, $type);
         }
     }
-    log_warning("Ignoring WS message with unknown type $type from $webui_host:\n" . pp($json));
-}
-
-sub _handle_command_info {
-    my ($json, $client, $worker, $webui_host) = @_;
-
-    if (my $population = $json->{population}) {
-        $client->webui_host_population($population);
-    }
 
-    # enfore recalculation of interval for overall worker status update
-    $client->send_status_interval(undef);
+    # WS messages of type "info" can be sent for debugging
+    log_warning("Ignoring WS message with unknown type $type from $webui_host:\n" . pp($json)) if $type ne 'info';
```

Comment:
So currently with your change above we would see this warning for every status update? Maybe make it a `log_debug`?

Reply:
That's why i appended the `if $type ne 'info'`. We will not be seeing those in production. 😉

Reply:
oooh. I reversed the logic in my head...